package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "searcher",
+  "version": "0.1.0",
+  "private": true,
+  "env": {
+    "browser": true,
+    "commonjs": true,
+    "es6": true,
+    "jquery": true
+  },
+  "dependencies": {
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "react-infinite-scroller": "^1.0.12"
+  },
+  "devDependencies": {
+    "elasticsearch": "^13.0.1",
+    "gh-pages": "^1.0.0",
+    "react-scripts": "1.0.5"
+  },
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build",
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "homepage": "http://rohittm.github.io/Searcher"
```

Comment:
I am not sure what's right way, please figure out what do people put in homepage in this kind of set up (fork --> PR)?